README.md

```diff
@@ -21,18 +35,59 @@ Just make sure your changes and pull requests are in accordance with the [contri
 
 We are not currently accepting translations of the game on our main repository. If you would like to translate the game into another language consider creating a fork or contributing to a fork.
 
+## Project activity
+<!---
+CHANGE THE API!!!! API can be obtained on the website repobeats.axiom.co | HERE, FOR EXAMPLE, THERE IS AN API FROM ANOTHER REPOSITORY
+--->
+![Project activity](https://repobeats.axiom.co/api/embed/4637fb51923408d570b8e555b3fde24eedb2bfea.svg 'Repobeats analytics image')
+
+## Project contributors
+
+List of people who contributed to the project:
+
+[![Contributors](https://contrib.rocks/image?repo=space-wizards/space-station-14)](https://github.com/space-wizards/space-station-14/graphs/contributors)
+
 ## Building
 
-1. Clone this repo.
-2. Run `RUN_THIS.py` to init submodules and download the engine.
-3. Compile the solution.
+**Clone** this repo:
+```shell
+# Cloning using HTTPS repository access
+git clone https://github.com/{user}/space-station-14.git
+
+# Cloning using SSH repository access
+git clone git@github.com:{user}/space-station-14.git
+```
+
+> [!NOTE]
+> Replace `{user}` to your GitHub username. The easiest way to find it is in the GitHub service at the very top of the repository page.
+
+**Go** to the project folder and run `RUN_THIS.py` to initialize the sub-modules and load the engine:
+```shell
+cd space-station-14
+python RUN_THIS.py
+```
+Compile the solution: 
+
+Build the server and use `dotnet build` or `dotnet build --configuration Release` (if necessary).
```

Comment:
IMO you should make this part try to redirect people to the docs instead, as all useful information to anyone new will be here. And this will just be a really condensed version to get the build running.

1. Numbered list
2. No need to tell someone how to clone with ssh
3. How can someone reading this clone this repo with their own name? We have not told them to fork. Just use our own repo here.
4. Unnecessary bolding.
5. Unnecessary release building instruction.